core/modules/layout/layout.theme.inc

```diff
@@ -228,19 +228,35 @@ function theme_layout_settings_context_table($variables) {
 }
 
 /**
- * Output the list of visibility conditions for layouts.
+ * Output a list of items for layouts.
  */
-function theme_layout_conditions($variables) {
+function theme_layout_list($variables) {
   $element = $variables['element'];
+  $attributes = array();
+  if (isset($element['#attributes'])) {
+    $attributes = $element['#attributes'];
+  }
 
-  $output = '<ul>';
+  $output = '<ul ' . backdrop_attributes($attributes) . '>';
   foreach (element_children($element) as $key) {
     $output .= '<li>' . backdrop_render($element[$key]) . '</li>';
   }
   $output .= '</ul>';
   return $output;
 }
 
+
+/**
+ * Output the list of visibility conditions for layouts.
+ *
+ * @deprecated since 1.19.0.
+ */
+function theme_layout_conditions($variables) {
```

Comment:
If we're deprecating this theme function, shouldn't we stop calling it from places in core? But then it wouldn't be used anywhere... so perhaps just remove it?

Alternatively, we could leave this function as-is and add `theme_layout_list()` (or something like `theme_layout_action_links()`) and put a `@todo` in there to merge the two functions in 2.x?

Reply:
Okay, I changed the name to `theme_layout_action_links()` and left it separate from conditions. You're right that conditions are special, with their operation links in the `<LI>`. I agree that they should probably not share the same function now, just because the function seems the same, we might want to replace those with something different in the future.